### ISSUE TITLE
Does adanet support GBDT as subnetwork?

### ISSUE THREAD

**Issue by fangkuann** [tensorflow/adanet#121]

**Comment by cweill** [comment]
@fangkuann: Yes it does! You can see how we use it [here](https://github.com/tensorflow/adanet/blob/master/adanet/core/estimator_distributed_test_runner.py#L316). Note this is the old `contrib` version of GBDT, and there is a new one called `tf.estimator.BoostedTreesEstimator` which should work, however we haven't tried it. 

Please give it a shot and let us know how it works for you.

**Comment by fangkuann** [comment]
@cweill  thanks for your answer!
In this paper https://ai.google/research/pubs/pub48133/, the authors combine the GBDT based model and DNN based model to enhance model performance. 
Therefore I want to reproduce it in the product where I worked for.  Since I want to training two model using only one training tool (Training GBDT using LightGBM and training NN using TensorFlow makes system complex)

According to the tutorial https://medium.com/tensorflow/combining-multiple-tensorflow-hub-modules-into-one-ensemble-network-with-adanet-56fa73588bb0.
It seems using BoostedTreesEstimator here is almost same as using LinearEstimator, just replace the linear_estimator instead of tree_estimator like code below.  

    import adanet
    estimator = adanet.AutoEnsembleEstimator(
        head=ranking_head,
        candidate_pool=[
            tree_estimator,
            dnn_estimator
        ],
        config=run_config,
        max_iteration_steps=50000,
    )

However, our DNN based model has already trained and deployed using TF-Ranking library.  Therefore, the input_fn for training has the shape of [None, list_size, 1]. When running, the GBDT throws an exception: 
`ValueError: List argument 'bucket_boundaries' to 'boosted_trees_bucketize' Op with length 1 must match length 64 of argument 'float_values'.`
where 64 is list_size in our training config.

I did not know whether it is still available to combine TF-Ranking and AdaNet.  If it is yes, what is the guideline to do it.
